### PR TITLE
Felix/mujoco datasets

### PR DESCRIPTION
**Pull request by fleibfried**
PLEASE DO NOT MERGE

### CODE REVIEW

File: bayesian_benchmarks/data.py [hunk 1]
```diff
@@ -753,6 +755,173 @@ class C(Classification):
         name, N, D, K = name, N, D, K
 
 
+path_to_mujoco_dataset = '/home/felix/Desktop/policy_destination'
```
Comment (vdutor) [inline bayesian_benchmarks/data.py]: I would set this to `None` and throw a warning saying a user should specify the path to the dataset.

Comment (vdutor) [reply bayesian_benchmarks/data.py]: Can you add a comment here saying that this is a temporary solution, and that we are working on making the data accessible from the internet.

File: bayesian_benchmarks/data.py [hunk 2]
```diff
@@ -753,6 +755,173 @@ class C(Classification):
...
+policy_checkpoints = [str(i * 100000) for i in range(11)]
+evaluation_suffix = 'sac_policy.eval'
+evaluations = 10
+
+
+class MujocoSoftActorCriticDataset(Dataset):
+    """
+    The dimensions of the observation space and the action space are given by
```
Comment (vdutor) [inline bayesian_benchmarks/data.py]: Can you add a bit more information about the dataset

**Review by vdutor — CHANGES_REQUESTED** [review]
Please add a couple of minor docstrings

File: bayesian_benchmarks/data.py (hunk 1 above)
Comment (fleibfried) [reply bayesian_benchmarks/data.py]: done

File: bayesian_benchmarks/data.py (hunk 2 above)
Comment (fleibfried) [reply bayesian_benchmarks/data.py]: Done

**Comment by fleibfried** [comment]
comments addressed

**Pull request merged**